### ISSUE TITLE
Finish the #608 phoneme-blending policy: four families still publish unpaired B results

### ISSUE THREAD

**Issue by frankbuckley** [dseinternational/language-reading-predictors#619]
> [!NOTE]
> Drafted by a LLM-based AI tool (Claude Code/Opus 5).

## Why this exists

The #608 decision (2026-08-24, `notes/202608242000-blending-guessing-floor-scope-608.md`) settled that the mandatory guessing-floor link sensitivity applies to **every** registered model with `outcome_symbol="B"`, in every family, under **one** archive-grade evidence tier. #608 was documentation-only and is closed; #596 implemented the gain family and is closed. Nothing open tracks the rest, so a settled scientific decision currently has **five live reporting fits out of compliance and no issue behind them**.

## Current state

`release._blending_pair_release_failures` still dispatches on `kind`:

```python
if kind == "level_factors": ...
if kind == "did":           ...
if kind == "gain_factors":  ...
if kind != "itt": return ()
```

So four families fall straight through. Verified against the stored `reporting` fits on `main` (`release.evaluate_publication`):

| Model | Family | Below-chance mass | `release_decision` today |
| --- | --- | ---: | --- |
| `lrp-rli-al-006` | `aligned` | 4.9 % | `publishable=True`, unpaired |
| `lrp-rli-ca-007` | `concurrent` | 9.7 % | `publishable=True`, unpaired |
| `lrp-rli-dose-084` | `dose_response` | 7.0 % | `publishable=True`, unpaired |
| `lrp-rli-med-087` | `mediation` | 11.6 % | `publishable=True`, unpaired |
| `lrp-rli-med-187` | `mediation` | 11.6 % | `publishable=True`, unpaired |

**`lrp-rli-med-187` is not in the #608 note's list of eight.** It is a registered `B` model — the `interventional` relabelling of MED-087 — and carries the same below-chance mass, so the note undercounted. There are **16** registered `B` models, not 13.

None of these four families declares `score_mean_link` anywhere in its settings module, run plan or factory.

## Why it matters

The link is not a rescaling. On the two pairs now fitted under both links the natural-scale answer moves materially:

| | ordinary logit | guessing floor |
| --- | ---: | ---: |
| LRPITT08 / 08b, items | +0.99 | +0.49 |
| LRPGF06 / 06f, items | +0.84 | +0.50 |
| LRPGF06 / 06f, 89 % CI | +0.09 to +1.58 | −0.06 to +1.04 |
| LRPGF06 / 06f, P(benefit ≥ 1 item) | 0.359 | 0.071 |

In both pairs the estimate falls roughly 40 %, the interval crosses zero and the practical verdict inverts — and in both, PSIS-LOO marginally *favours* the floor link (ITT +1.09 SE 2.68; gain +2.49 SE 3.55). The constraint costs nothing in fit and is motivated by the test's own design, so it is the ordinary-link number that needs justifying.

#608 already rejected the argument that observational families are exempt because their quantity is an association rather than a contrast: any quantity reported on the natural scale inherits the link dependence regardless of what identifies it. `METHODS.md` defines every dose fit's focal estimand as a natural-scale marginal, so `dose_response` is squarely inside that.

### Per family — `aligned`, `concurrent`, `dose_response`, `mediation`

- [ ] Add `score_mean_link` to the family's typed settings, resolved run plan and factory, with the same B-only / graded-only contract and validation-before-I/O that `gain_factors` uses (#596).
- [ ] Audit every natural-scale summary the family's pipeline derives and map it through the link the factory **built**, read from the fitted payload rather than the declared setting. This is the failure mode that fails quietly: in the gain family `treatment_marginal_effect` and `association_marginals` differenced raw inverse-logits with no link parameter at all, and would have published ordinary-link items from a floor-link posterior.
- [ ] Check whether the family has an empirical-Bayes intercept anchor needing `invert_score_mean_link` (the level family's moved 1.1 logits; the gain family had none).
- [ ] Register the companion, decide and record its model id, and extend the release gate.
- [ ] Fit the companion at reporting tier — until both halves exist the pairing is nominal.

### Cross-cutting

- [ ] Key the gate on `outcome_symbol` rather than `kind`, so a **new** `B` model in any family fails closed by default rather than needing its family to be remembered. This is what #608 says the code already does; it does not.
- [ ] Decide the scope rule for non-model-of-record `B` fits and apply it consistently. Three models sit in this gap — `lrp-rli-gf-106`, `lrp-rli-gf-206` (exempt by dated record, #596) and `lrp-rli-lf-206` (exempt in shipped `level_factors.model_of_record_window` code) — while the #608 note lists all three as needing companions. They should move together, in whichever direction.
- [ ] Implement #608 decision 2: bind every pair by the content-addressed archive apparatus, not the two-directory stored-artefact check. Today only the ITT pair is archive-bound; `gain_factors`, `level_factors` and `did` use the lighter tier, and the rendered reports do not show a reader which tier backs a given card — which is the ambiguity the policy exists to remove. Needs `blending_sensitivity`'s archive parameterised by pair and by per-family focal columns (it is currently hardcoded to one global pair with ITT-shaped free variables and an ITT-shaped summary-column map).

### Prerequisites already known

- [ ] `lrp-rli-lf-106` and `lrp-rli-did-103` are registered but have **never been fitted at reporting tier**, so the level and DiD pairings are nominal and both `lrp-rli-lf-006` and `lrp-rli-did-003` currently evaluate `publishable=False`. `lrp-rli-lf-206` has no stored fit at all.

## Sequencing note

Adding `score_mean_link` to a family's settings changes the stored primary's resolved run plan even when the fitted model is unchanged. The gain pair was unaffected — the pair check compares data, fitted rows, sampling config and row count, not the plan digest — so the stored `lrp-rli-gf-006` fit stood as-is and only the companion needed fitting. Confirm the same for each family before assuming five fits rather than ten.

## References

- #608 — the decision (`notes/202608242000-blending-guessing-floor-scope-608.md`)
- #596 / #618 — the gain family's implementation and the scope precedent (`notes/202608251100-gain-blending-guessing-floor-596.md`)
- #584 decision 2 — the level pair; #576 finding 2 — the DiD pair
- #575 acceptance item 7 — closed by #618 for the gain family
